DogsList/.swiftlint.yml

```diff
@@ -3,10 +3,6 @@ disabled_rules: # rule identifiers turned on by default to exclude from running
   - colon
```

Comment:
do not disable swiftlint rules
it's not a good practice
enable all rules and update your code to comply the linter rules

Reply:
Fixed.